README.md

```diff
@@ -10,9 +10,7 @@ Jakarta EE is the future of cloud native Java. Jakarta EE open source software d
 
 Install dependencies, build assets and start a webserver:
 
-```bash
-npm install && hugo server
-```
+npm install && npm run generate-projects && npm run dev && hugo server --disableFastRender --buildFuture
```

Comment:
I think we should run the generate-projects in the postinstall script:
https://docs.npmjs.com/misc/scripts

This would reduce the number of commands that will be required to generate the website.

npm run dev is not required.